rllib/models/modelv2.py

```diff
@@ -189,12 +192,16 @@ def __call__(
             state (list): list of state tensors with sizes matching those
                 returned by get_initial_state + the batch dimension
             seq_lens (Tensor): 1d tensor holding input sequence lengths
+            return_values (bool): Whether to also return the values tensor.
+                If True, will return 3 items: [output, states, values], instead
+                of 2.
 
         Returns:
-            (outputs, state): The model output tensor of size
+            (outputs, state, values?): The model output tensor of size
```

Comment:
I'd suggest leaving the shape of the tuple fixed, and either populate values or not based on that flag (maybe require the user to return values=None if flag is set to false?)